docs/.vuepress/config.js

```diff
@@ -13,11 +13,107 @@ const sidebar = {
     {
       collapsable: false,
       title: '⚙️ Setup & Deployment',
+      sidebarDepth: 0,
       children: [
-        ['/developer-docs/latest/setup-deployment-guides/installation.md', 'Installation'],
+        {
+          title: 'Installation',
```

Comment:
Do we really need this one expanded by default?

Reply:
I think we should close it, the same as the plugin section.
otherwise, it LGTM.
Great job @pwizla

Reply:
Thank you for the feedback @derrickmehaffy and @lauriejim :-)

I [fixed it](https://github.com/strapi/documentation/pull/235/commits/dcac8e8ee1ba9d22313d748251de3d271fd6cfef)